File: CineMatic.Model/Izvjestaji/TopKorisnik.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CineMatic.Model.Izvjestaji
{
    public class TopKorisnik
    {
        public int? Korisnikd { get; set; }
        public string? Ime { get; set; }
        public string? Prezime { get; set; }
        public decimal? UkupnoPotrosenoNovca { get; set; }
    }
}

File: UI/cinematic_desktop/lib/models/top_korisnik.dart
class TopKorisnik {
  final int korisnikId;
  final String ime;
  final String prezime;
  final double ukupnoPotrosenoNovca;

  TopKorisnik({
    required this.korisnikId,
    required this.ime,
    required this.prezime,
    required this.ukupnoPotrosenoNovca,
  });

  factory TopKorisnik.fromJson(Map<String, dynamic> json) => TopKorisnik(
        korisnikId: json['korisnikd'] ?? json['korisnikId'] ?? 0,
        ime: json['ime'] ?? '',
        prezime: json['prezime'] ?? '',
        ukupnoPotrosenoNovca: (json['ukupnoPotrosenoNovca'] ?? 0).toDouble(),
      );
}